resolving/git-resolver/test/index.ts

```diff
@@ -414,11 +414,9 @@ aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa\trefs/heads/master\
   const resolveResult = await resolveFromGit({ pref: 'git+https://0000000000000000000000000000000000000000:x-oauth-basic@github.com/foo/bar.git' })
   expect(resolveResult).toStrictEqual({
     id: '0000000000000000000000000000000000000000+x-oauth-basic@github.com/foo/bar/0000000000000000000000000000000000000000',
-    normalizedPref: 'git+https://0000000000000000000000000000000000000000:x-oauth-basic@github.com/foo/bar.git',
+    normalizedPref: 'github:foo/bar',
     resolution: {
-      commit: '0000000000000000000000000000000000000000',
-      repo: 'https://0000000000000000000000000000000000000000:x-oauth-basic@github.com/foo/bar.git',
-      type: 'git',
+      tarball: 'https://codeload.github.com/foo/bar/tar.gz/0000000000000000000000000000000000000000',
     },
     resolvedVia: 'git-repository',
```

Comment:
but where is the auth token? This test is about a private git-hosted dependency that can only be downloaded with an auth token. But you have removed the auth token from the resolveResult.

Reply:
Fixing this bug involved more than I had imagine, I'll look into this next week.

Reply:
@zkochan It was `HostedGit.shortcut()` that removed the auth token.